site/client/owid.entry.ts

```diff
@@ -57,6 +57,7 @@ new SmoothScroll('a[href*="#"][data-smooth-scroll]', {
 
 const search = document.querySelector("form#search-nav") as HTMLFormElement
 if (search) {
+    // Todo: Not seeing this event in Amplitude. Can this be removed?
```

Comment:
Ah yes, it can be removed, or we can fix the selector to match the current markup. I think it should be `form.HeaderSearch` to target the header search box.

Reply:
I will remove.

I noticed we have a little bit of search tracking now in Amplitude via OWID_PAGE_LOAD with an href filter. I started a chart and dashboard for that. That excludes instant searches though.

We could move this tracking event to `HeaderSearch.tsx`. I think it's probably best for now to stick to Algolia for tracking search queries, instead of duplicating that info into Amplitude.